lib/active_merchant/billing/gateways/adyen.rb

```diff
@@ -870,11 +871,11 @@ def cvv_result_from(response)
       def endpoint(action)
         case action
         when 'disable', 'storeToken'
-          "Recurring/#{RECURRING_API_VERSION}/#{action}"
```

Comment:
Can you still please add a adyen unit test to very the version is being properly passed? I think adding one unit test to all the gateways would be good

Reply:
I added tests to check that the URLs are being properly build. For the gateways that don't use the API version (braintree and hps), I'm not sure if it's worth adding a test. Let me know what you think of this approach.